academy/4-my-own-chain/game-forfeit.md

```diff
@@ -246,7 +246,7 @@ $ checkersd tx checkers play-move 2 0 5 1 4 --from $alice
 
 ---
 
-Be sure to space each `tx` command from a given account by a couple of seconds so that they each go into a different block - `checkersd` is limited by the fact that it uses the account's transaction sequence number by fetching it from the current state. With three games in, confirm that you see them all:
+Space each `tx` command from a given account by a couple of seconds so that they each go into a different block - `checkersd` is limited because it uses the account's transaction sequence number by fetching it from the current state. With three games in, confirm that you see them all:
```

Comment:
```suggestion
Space each `tx` command from a given account by a couple of seconds so that they each go into a different block - by default `checkersd` is limited because it uses the account's transaction sequence number by fetching it from the current state.

<HighlightBox type="tip">

 If you want to overcome this limitation, look at `checkersd`'s `--sequence` flag:
 
 ```sh
 $ checkersd tx checkers create-game --help
 ```sh

And at your account's current sequence. For instance:

```sh
$ checkersd query account $alice --output json | jq ".sequence"
```sh

Which returns something like:

```json
"9"
```json

</HighlightBox>

With three games in, confirm that you see them all:
```
@AndrewB9lab You know, before adding my suggestion to a batch, you can edit it too.

Reply:
So, if I use **Edit** in these comment windows, changes I make inside the ```suggestion``` tags will be committed to the MD, correct? Thank you, that's good to know.

Reply:
If you edit the suggestion in my comment, then the comment and the suggestion change. After that, you are back to a suggestion that is here waiting to be committed or not.